dashboards/hadoop/hadoop-prometheus-overview.json

```
@@ -86,25 +97,31 @@
             },
             "dataSets": [
               {
+                "breakdowns": [],
+                "dimensions": [],
+                "measures": [],
                 "plotType": "LINE",
                 "targetAxis": "Y1",
                 "timeSeriesQuery": {
-                  "prometheusQuery": "hadoop_namenode_capacityused"
+                  "prometheusQuery": "hadoop_namenode_capacityused",
```

```suggestion
                  "prometheusQuery": "hadoop_namenode_capacityused{${Cluster},${Location},${Namespace}}",
```

Should this also have the template variables applied to it?